website/content/docs/install/index.mdx

```diff
@@ -56,14 +56,44 @@ a copy of [`git`](https://www.git-scm.com/) in your `PATH`.
    $ make tools
    ```
 
-1. Build Consul for your current system and put the binary in `./bin/`
-   (relative to the git checkout). The `make dev` target is just a shortcut that
-   builds `consul` for only your local build environment (no cross-compiled
-   targets).
+1. Build Consul for your target system. The binary will be placed in `./bin`
+   (relative to the git checkout).
 
-   ```shell
-   $ make dev
-   ```
+<!--
```

Comment:
Not sure there's a better way around the problem described here

Reply:
I've run into this same problem with the pre-commit hook. `git commit --no-verify` will allow you to commit the content unmodified.

Reply:
Thanks Blake. Didn't know that was an option!